### ISSUE TITLE
"Consistent" passage of the plot?...

### ISSUE THREAD

**Issue by unihorn78** [Ellypse/Storyline#43]
Is it possible to implement a "consistent" passage of the plot?...

Now the emergence of a new quest is tied to the level of the player.

And often arises the situation when quest, formally, is continuation of previous, but can be fulfilled until him (until previous).

And it turns out " anomaly of cause and effect"

So here is realistically whether add option which (for example focusing on level of quest, and list quests completed player) will not allow obtain "high-level" quest until low-level (especially as relevant if you have outgrown level of locations).

For example until you not complete all "low-level quests locations" high-level obtain not you can (they not will show up in Storyline).

Similarly with locations. For obtain quests locations high-level need to do quests locations low levels.

With locations there can be two ways.

SIMPLE. If locations one level several (especially this concerns "classical world" until 60 levels). The choice of one of them, automatically disables the possibility of taking quests in other locations of comparable level (it turns out a full "variability" of the passage). Accordingly, to correct and "Board orders" (letting them through the StoryLine (or simply disabling their frame and replacing it with some "neutral messages")).

To process only the quests needed for the achievement "Loremaster". A normal display as usual...

That will allow you to move around the world as you like (and how you want to take quests) without risk "to break the sequence of history."

The method is perfect for the preliminary implementation of the idea.

COMPLEX. Assess the chain of the plot on a group of locations. And proceed from it...

***

As additional options (if possible)...

If you have not completed the story chain, but your level exceeded the level of the ocean, and she entered the phase of "new realities" (plot BfA for example:)) then, for example, show a pop-up message that, for example, "looks like you fell into a temporary anomaly; find dragon time; he's somewhere in this area; it will allow you to return to your time" (to check for quests, the current level of the player, and location).

Perhaps, so, can be solve and some roughness (say living Varrian Rin in Northrend; under his tomb in Stormwind; or books Pro of slain Deathwing in Stormwind, and quests with his the participation (quest a dragon-Goblin conducting experiments with Black Dragons)).

For example, by default, these quests are disabled. And on arrival in Northrend, or Pandaria, show a pop-up sign, that, for example, "that you have been contacted by the dragons of time; and warned that for your further journey you need to travel to the past of this land."..

***

If it is possible.... That would be a dream...

PS.

Sorry for the bad English...

**Comment by Ellypse** [comment]
Storyline is just a quest frame replacement, reformatting the information available to you. It has no vocation to guide or influence the gameplay and story telling. What you are asking for is out of the scope of my goal for this add-on.

**Comment by unihorn78** [comment]
Goals can always be supplemented. Even if not immediately, " this minute".

And a separate, own, window quest it simply created by for this...

I would very much like something you would at least have considered the possibility to implement at least the "simple" method...

That, if technically possible, will not harm the addon (especially if we are talking about the "dive into history" and roleplay). 

With Respect.

**Comment by Zarggg** [comment]
What you're looking for is more like a quest guide addon.

**Comment by unihorn78** [comment]
> What you're looking for is more like a quest guide addon.

I'm not looking for a guide for quests (such as, for example, "Wholly" (recommended for beginners)).

I've been playing for years. And know where, what, quests (and as their undergo, and in what sequence) and without guides this kind of.

What I'm looking for, I described in detail.

I seek not information "in what sequence, correctly, undergo the plot" (on their own managing "cause of and consequence of").

I'm looking for a way to do so, that-would, in the plot, I was led by the game itself (physically not giving me a postquest, before the prequest).

Not me.

I'm not God, and I'm not even a Cromie.

I'm looking for a way to just play. And not to engage in "temporal micromanagement", making sure that, accidentally, at first, not to "fly into space", and only then invent a "space rocket" (which will make this flight possible).

The approximate algorithm for ensuring this I described in this sentence (independent quest window is ideal for this: it does not affect the program code of the game, and is an add-on to it). But, unfortunately, the author of this, excellent, addon refused to even consider the possibility of such...

With Respect.
